content/intro-to-storybook/vue/fr/using-addons.md

```diff
@@ -0,0 +1,172 @@
+  <source
+    src="/intro-to-storybook/addon-knobs-demo.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### Installation
+
+Tout d'abord, nous devrons l'ajouter en tant que dépendance.
+
+```bash
+yarn add -D @storybook/addon-knobs
+```
+
+Enregistrez Knobs dans votre fichier `.storybook/main.js`.
+
+```javascript
+// .storybook/main.js
+
+module.exports = {
+  stories: ['../src/components/**/*.stories.js'],
+  addons: ['@storybook/addon-actions', '@storybook/addon-knobs', '@storybook/addon-links'],
+};
+```
+
+<div class="aside">
+<strong>📝 L'ordre d'enregistrement des addons est important!</strong>
+<br/>
+L'ordre dans lequel vous listez ces addons dictera l'ordre dans lequel ils apparaissent sous forme d'onglets sur votre panneau d'extension (pour ceux qui y apparaissent).
+</div>
+
+C'est tout! Il est temps de l'utiliser dans une histoire.
+
+### Utilisation
+
+Utilisons le type de bouton d'objet dans le composant `Task`.
+
+Tout d'abord, importez le décorateur `withKnobs` et l'`objet` knob dans `Task.stories.js`:
+
+```javascript
+// src/components/Task.stories.js
+import { action } from '@storybook/addon-actions';
+import { withKnobs, object } from '@storybook/addon-knobs';
+```
+
+Ensuite, dans l'exportation par `défaut` du fichier `Task.stories`, ajoutez `withKnobs` à la clé `decorators`:
+
+```javascript
+// src/components/Task.stories.js
+
+export default {
+  title: 'Task',
+  decorators: [withKnobs],
+  // pareil qu'avant
+};
+```
+
+Enfin, intégrez l'`objet` know dans l'histoire "par défaut":
+
+```javascript
+// src/components/Task.stories.js
+
+// état de la tâche par défaut
+export const Default = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: object('task', { ...taskData }),
+    },
+  },
+  methods: actionsData,
+});
+
+// pareil qu'avant
+```
+
+Maintenant, un nouvel onglet "Knobs" devrait apparaître à côté de l'onglet "Action Logger" dans le volet du bas.
+
+Comme documenté [ici](https://github.com/storybooks/storybook/tree/master/addons/knobs#object), l'`objet` know accepte une étiquette et un objet par défaut comme paramètres. L'étiquette est constante et apparaît à gauche d'un champ de texte dans votre panneau de modules complémentaires. L'objet que vous avez transmis sera représenté sous la forme d'un objet blob JSON modifiable. Tant que vous soumettez un JSON valide, votre composant s'ajustera en fonction des données transmises à l'objet!
+
+## Les modules complémentaires font évoluer la portée de votre livre de contes
+
+Non seulement votre instance Storybook sert d'[environnement CDD](https://blog.hichroma.com/component-driven-development-ce1109d56c8e) merveilleux, mais nous fournissons maintenant une source de documentation interactive. Les PropTypes sont excellents, mais un concepteur ou quelqu'un de complètement nouveau dans le code d'un composant sera capable de comprendre son comportement très rapidement via Storybook avec l'addon de boutons implémenté.
```

Comment:
@jeanphibaconnais when you're able please adjust the link "environnement CDD" to point at https://www.componentdriven.org/

Reply:
it's modified 😄